### ISSUE TITLE
Support for Invisible Recaptcha

### ISSUE THREAD

**Issue by slinkymanbyday** [praekelt/django-recaptcha#120]
Any plans to support the Invisible Recaptcha (currently in Beta)?
https://www.google.com/recaptcha/intro/comingsoon/invisiblebeta.html

**Comment by adamchainz** [comment]
Not personally, I am only a maintainer on this repo. But feel free to make a PR when it's out...

**Comment by MertcanGokgoz** [comment]
Yes this new Invisible Recaptcha support should come, I am waiting this

**Comment by ka1nsha** [comment]
I want this

**Comment by Pringels** [comment]
I will look into it

**Comment by kz26** [comment]
Shouldn't be too difficult - the only challenge I see is that you need to bind an event handler to either the form's submit button or the form onSubmit self, which is slightly tricky in cases like this where you're trying to build a generic plugin for a web framework.

**Comment by slinkymanbyday** [comment]
@kz26, maybe if require to pass in the submit button id as a parameter.
something like {% recaptcha 'submit-id' %}

**Comment by kz26** [comment]
I've added Invisible Recaptcha support to my fork of the older django-nocaptcha-recaptcha project here: https://github.com/kz26/django-nocaptcha-recaptcha

It would've probably been more helpful to add support to django-recaptcha, but I already had a project using this library, and the architecture of django-recaptcha doesn't seem to allow switching on-the-fly between different types of reCAPTCHA easily.

**Comment by robwise1** [comment]
Any updates on this?

**Comment by AltusBarry** [comment]
reCAPTCHA V2 Invisible support was added in django-recaptcha version 1.5.0, V3 support was added in django-recaptcha version 2.0.0
